lib/cbra/cli.rb

```diff
@@ -14,5 +14,11 @@ def ls(app_path)
     def version
       puts Cbra::VERSION
     end
+
+    desc "graph APP_PATH", "Outputs graph.png to the current directory"
+    def graph(app_path)
+      Graph.new(app_path).png
+      puts "Graph generated in root directory"
```

Comment:
@benlangfeld - Then "Graph generated in current directory" ? It's generated wherever this command is run, totally independent of the passed `app_path`.